src/main/scala/org/hammerlab/guacamole/commands/jointcaller/SomaticJointCaller.scala

```diff
@@ -185,35 +186,38 @@ object SomaticJoint {
 
     assume(loci.nonEmpty)
 
-    // When mapping over pileups, at locus x we call variants at locus x + 1. Therefore we subtract 1 from the user-
-    // specified loci.
-    val broadcastForceCallLoci = sc.broadcast(forceCallLoci)
-
     val lociPartitions =
       args
         .getPartitioner(readsets.allMappedReads)
         // When mapping over pileups, at locus x we call variants at locus x + 1. Therefore we subtract 1 from the user-
         // specified loci.
         .partition(lociSetMinusOne(loci))
 
+    val broadcastForceCallLoci = sc.broadcast(forceCallLoci)
+
+    def callPileups(pileups: PerSample[Pileup]): Iterator[MultiSampleMultiAlleleEvidence] = {
+      val forceCall =
+        broadcastForceCallLoci
+          .value
+          .onContig(pileups.head.contigName)
+          .contains(pileups.head.locus + 1)
+
+      MultiSampleMultiAlleleEvidence.make(
+        pileups,
+        inputs,
+        parameters,
+        reference,
+        forceCall,
+        onlySomatic,
+        includeFiltered
+      ).toIterator
+    }
+
     pileupFlatMapMultipleRDDs(
       readsets.mappedReads,
       lociPartitions,
       skipEmpty = true,  // TODO: shouldn't skip empty positions if we might force call them. Need an efficient way to handle this.
-      rawPileups => {
```

Comment:
factored this code out to a callback so that this `pileupFlatMapMultipleRDDs` call can be a little less gnarly